packages/stark-ui/assets/styles/_base.scss

```diff
@@ -139,3 +126,63 @@ td {
     padding-right: 15px;
   }
 }
+
+/* stark session ui pages */
+.stark-session-ui-page {
+  position: fixed;
```

Comment:
For consistency, I think we should use flexbox here to center the content

Reply:
This will be tackled in #718